### PR TITLE
Update fixed scripts to include recent changes

### PR DESCRIPTION
**Pull request by morningspace**
This is to address #288 and many other similar issues people reported here and there. Just the outputs generated by running `genfixed.sh` in master branch to pickup some recent changes, especially #271.

Give that quite a few people are still using the fixed scripts according to the repo doc and the released scripts haven't been updated yet. (Maybe could someday in next release:-)), I'd propose to update these fixed scripts so as to make some people lives easier...

Feel free to take review and merge if that makes sense...

### CODE REVIEW

**Comment by morningspace** [comment]
All tests passed! FYI @ivan4th

**Comment by morningspace** [comment]
Hi @ivan4th, and others, just curious any other comments regarding getting this merged... understood the fixed scripts appear to be deprecated... really hope that's merged officially so that can unblock some people's pain on using that...

**Comment by morningspace** [comment]
Any comments? @ivan4th @jellonek @roberthbailey

File: fixed/dind-cluster-stable.sh [hunk 1]
```diff
@@ -524,10 +524,14 @@ DIND_NO_PROXY="${DIND_NO_PROXY:-}"
 DIND_DAEMON_JSON_FILE="${DIND_DAEMON_JSON_FILE:-/etc/docker/daemon.json}"  # can be set to /dev/null
 DIND_REGISTRY_MIRROR="${DIND_REGISTRY_MIRROR:-}"  # plain string format
 DIND_INSECURE_REGISTRIES="${DIND_INSECURE_REGISTRIES:-}"  # json list format
+# Comma-separated custom network(s) for cluster nodes to join
```
Comment (jellonek) [inline fixed/dind-cluster-stable.sh]: Rest of comments (e.g. above, and the same comment in other files) is lowercase started, looks like this was manually uppercased by accident, right?

File: fixed/dind-cluster-stable.sh [hunk 2]
```diff
@@ -1057,6 +1059,14 @@ function dind::run {
          ${opts[@]+"${opts[@]}"} \
          "${DIND_IMAGE}" \
          ${args[@]+"${args[@]}"}
+
+  if [[ -n ${DIND_CUSTOM_NETWORKS} ]]; then
+    local cust_nets
+    local IFS=','; read -ra cust_nets <<< "${DIND_CUSTOM_NETWORKS}"
```
Comment (jellonek) [inline fixed/dind-cluster-stable.sh]: Can you do that as in https://github.com/kubernetes-sigs/kubeadm-dind-cluster/blob/b4554ac24db0480626372e6382bbd24e4d280229/dind-cluster.sh#L129 ?

**Review by jellonek — CHANGES_REQUESTED** [review]
IMO that looks quite ok. Only two minor things found.

**Comment by jellonek** [comment]
For me it looks fine now. @ivan4th can you take a look?

**Comment by morningspace** [comment]
Hey @jellonek thanks for your prompt feedback! Just happen to notice your last comment a few mins ago :-) And, @ivan4th look forward your comments as well... Thx.

P.S.
I saw some of the tests run on Circle CI will fail occasionally due to an error "Address already in use". In my case, it's `test_src_master`. Did a quick check and it appears when we start port forwarding using socat, the port number has been used by other process on the same machine already. In my case, it's `0.0.0.0: 32768`. I am not sure how Circle CI run jobs and who else is using 32768. But it seems very likely to me that Circle CI is using pooled machine which has run kubeadm-dind-cluster tests previously, and it looks we do not kill the socat process after the current test is finished. Correct me if any.

Anyway, I'd say that's not what this PR is trying to address. Maybe I can create another PR to solve it.

File: fixed/dind-cluster-stable.sh (hunk 1 above)
Comment (morningspace) [reply fixed/dind-cluster-stable.sh]: Yeah... now I turn it back to lowercase as I realized that it looks other comments are all lowercased... just to be consistent.

**Review by jellonek — APPROVED** [review]

**Comment by mattmb** [comment]
Is this ready to merge? I just had to rediscover/remember https://github.com/kubernetes-sigs/kubeadm-dind-cluster/pull/263 / https://github.com/kubernetes-sigs/kubeadm-dind-cluster/pull/266 from November because the fix still isn't released.

**Comment by morningspace** [comment]
Yes, it's ready for merge. @ivan4th Could you please help merge? Or, anyone else has permission to merge it?

**Comment by cofyc** [comment]
does this fix https://github.com/kubernetes-sigs/kubeadm-dind-cluster/issues/280?

**Comment by mrbobbytables** [comment]
Closing as this project is being deprecated and retired in favor of [KinD](https://github.com/kubernetes-sigs/kind).

Please see this issue for more details:
https://github.com/kubernetes/org/issues/1016

/close

**Pull request closed without merging**